tests/test_types.py

```diff
@@ -46,18 +46,34 @@ def test_item_type():
     assert 'invalid choice: x' in str(e.value)
 
 
-@mock.patch('planet.scripts.types.get_asset_types',
-            new=mock.Mock(return_value=DEFAULT_ASSET_TYPES))
-def test_asset_type():
-    check = convert_asserter(AssetType())
```

Comment:
The tests for v1 of this package very often use assertion helpers like this convert_asserter. It is unnecessary and the tests are easier to read and comprehend without it. Let's not do this in future work.